Gemfile

```diff
@@ -80,7 +80,7 @@ gem 'browser'
 # bundle config local.cmr_metadata_preview /path/to/local/git/repository
 # make sure to delete the local config when done making changes to merge into master
 # bundle config --delete local.cmr_metadata_preview
-gem 'cmr_metadata_preview', git: 'https://git.earthdata.nasa.gov/scm/cmr/cmr_metadata_preview.git', ref: '40320c29933'
+gem 'cmr_metadata_preview', git: 'https://git.earthdata.nasa.gov/scm/cmr/cmr_metadata_preview.git', branch: 'MMT-2366'
```

Comment:
I can't remember if the `branch:` needs to be changed or not, but intuition tells me yes

Reply:
It does, but I need to get the preview gem changes merged before I can make that change: https://git.earthdata.nasa.gov/projects/CMR/repos/cmr_metadata_preview/pull-requests/104/overview